source/use_case_climate_projection.rst

```diff
@@ -0,0 +1,176 @@
+
+  * do they comply with the FAIR principles?
+  * are there any bottlenecks?
+  * what can be done to remove the bottlenecks?
+
+Producers
+"""""""""
+
+The Norwegian climate service centre and partners (MET, NVE, NORCE, Bjerknessenteret) will produce the new predictions for the average and extreme air temperatures in Norway, forward in time to year 2100.
+
+.. uml:: air_temperature_projection_producer.puml
+
+Consumers
+"""""""""
+
+End users of the data might include: 
+
+* A researcher that wants to use the data in their biology model to predict the effect on ticks. 
+* A state agency that wants to investigate the effects temperature changes will have in each Norwegian kommune.
+* A Journalist that wants write about the potential changes that Norway will experience. 
+
+.. uml:: users_klima_leveranse_og_bruker.puml
+
+Trigger
+-------
+
+.. Event that initiates the Use Case (an external business event, a system event, or the first step
+   in the normal flow.
+
+Norwegian Environment Agency orders the report - Climate in Norway 2100. 
+The climate model data is needed to write this report. 
+
+
+Pre-conditions
+--------------
+
+.. Activities that must take place, or any conditions that must be true, before the Use Case can be
+   started.
+
+The following datasets must be available to the producer:
+
+* Bias adjusted time series from EURO-COREX (CMIP5 og CMIP6).
+* Calculated climate indexes, averages over certain time periods, etc.
+* Gridded historical weather observations.
+* Calculations of averages for fylke, kommune, and other regions.
+
+The following services should be used to present results:
+
+* OGC Web Map Service (WMS)
+* A solution for getting data
+
+Post-conditions
+---------------
+
+.. The state of the system at the conclusion of the Use Case execution.
+
+* A dataset is created that contains climate predictions forward in time until 2100.
+* The data should be stored on netcdf on a 1x1 km grid (follow-up: why netcdf?)
+* This dataset includes predictions about temperature, as well as many other variables.
+* The results of this dataset are distilled into a report (e.g., pdf) - Climate in Norway 2100.
+
+Normal Flow
+-----------
+
+.. Detailed description of the user actions and system responses that will take place during
+   execution of the Use Case under normal, expected conditions. This dialog sequence will ultimately
+   lead to accomplishing the goal stated in the Use Case name and description.
+
+* The researcher investigates the data
+
+  * downloads the climate predictions for temperature for the whole time range.
+  * downloads the climate prediction medians as an average over the period 2071-2100.
+
+* The journalist choses parameters on a website to se what the extreme temperatures will be in 2041-2070.
+* The state agency updates their maps with expected changes, to reflect future changes in water flow.
```

Comment:
we should change the decision here - how can a decision be relevant to changing temperature?

Reply:
Temperature effects planting conditions ... maybe something about farms and tree planting for wood?

Reply:
I tried to link it to permafrost - I'll push a new commit soon